packages/dmn-js-decision-table/src/Viewer.js

```diff
@@ -190,11 +190,11 @@ export default class Viewer extends Table {
       coreModule,
       TranslateModule,
       decisionTableHeadModule,
-      decisionTablePropertiesModule,
       decisionRuleIndicesModule,
       decisoinRulesModule,
       hitPolicyModule,
-      viewDrdModule
+      viewDrdModule,
+      decisionTablePropertiesModule
```

Comment:
Is it important to move this as the last module?

Reply:
It is important to load decision table properties after view DRD module so that we can maintain structure below:
```
view DRD button
decision table header (decision name and hit policy)
decision table
```
Otherwise, we would have to play with priorities.

Reply:
We already have priorities built-in: `onGetComponent(type, priority, callback) { ... }`. I'd rather use priorities instead of relying on a specific order of subscribing.

Reply:
Let's do that then.